code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1384,6 +1395,34 @@ var/list/rank_prefix = list(\
 					break
 	return 1
 
+/mob/living/carbon/human/proc/resuscitate_notify(type)
+	if(prob(50))
+		return
+	visible_message(SPAN_WARNING("\The [src] twitches and twists intensely"))
+	for(var/mob/O in viewers(world.view, src.loc))
+		if(O == src)
+			continue
+		if(!Adjacent(O))
+			continue
+		var/bio_stat = 0
+		if(O.stats)
+			bio_stat = O.stats.getStat(STAT_BIO)
+
+		if(bio_stat >= STAT_LEVEL_ADEPT)
+			switch(type)
+				if(1) //brain and heart fail
+					to_chat(O, "<font color='blue'>You can identify that [src]'s circulatory and central neural systems are failing, preventing them from resurrection.</font>")
+				if(2) //heart fail
+					to_chat(O, "<font color='blue'>You can identify that [src]'s circulatory system is unable to restart in this state.</font>")
+				if(3) //brain fail
+					to_chat(O, "<font color='blue'>You can identify that [src]'s central neural system is too damaged to be resurrected.</font>")
+				if(4) //corpse is too old
+					to_chat(O, "<font color='blue'>You see that rotting process in [src]'s body already gone too far. This is nothing but a corpse now.</font>")
+				if(5) //too much damage
+					to_chat(O, "<font color='blue'>[src]'s body is too damaged to sustain life.</font>")
```

Comment:
How all of this looks to ghosts? Can you make so that the text is all on a single line?

Reply:
Ghosts do not have stats as a module so they only see that they don't understand a thing

Reply:
Perhaps change that? Ghosts should be omnipotent. If someone LOOCs this info they are to eat shit.